ios/FluentUI/Command Bar/CommandBarButtonGroupView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class CommandBarButtonGroupView: UIView {
```

Comment:
are there 2 CommandBarButtonGroupView swift files and class?

Reply:
Yeah, must be an error during renaming. Deleted one.